sdcm/nemesis.py

```diff
@@ -6941,8 +6825,6 @@ def disrupt(self):
         self.disrupt_repair_streaming_err()
 
 
```

Comment:
The `UpgradeNemesisOneNode` is used in the `upgrade_schema_test.py` module.

But, that module is not used, so, since py code checkers don't complain we should be ok.

Reply:
Given that it has not been touched since basically 2017, can I remove it as well? I also do not see any usages of it

Reply:
Yes